libmproxy/flow.py

```diff
@@ -763,6 +764,29 @@ def tick(self, q, timeout):
     def duplicate_flow(self, f):
         return self.load_flow(f.copy())
 
+    def create_request(self, method, scheme, host, port, path):
+        """
+            this method creates a new artificial and minimalist request also adds it to flowlist
+        """        
+        c = ClientConnection.from_state(dict(
+                address=dict(address=(host, port), use_ipv6=False),
+                clientcert=None
+            ))
+
+        s = ServerConnection.from_state(dict(
+                address=dict(address=(host, port), use_ipv6=False),
+                state=[],
+                source_address=None, #source_address=dict(address=(host, port), use_ipv6=False),
+                cert=None
```

Comment:
Depending on the scheme, we should add `ssl_established=True` and `sni=host` here. Generally speaking I feel that it's not optimal depending on these attributes of the server connection, but I don't really know of a better place to put them.